exporter.go

```diff
@@ -162,3 +179,23 @@ func (e *exporter) Config() *config.Config {
 func (e *exporter) UpdateTarget(target []Target) {
 	e.targets = target
 }
+
+// registerSvcMetrics registers the metrics for the exporter itself.
+func registerSvcMetrics() *prometheus.CounterVec {
+	scrapeErrors := prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "scrape_errors",
```

Comment:
Great feature addition!
But shouldn't the metric be suffixed with `_total` given that it is a unitless counter?

Ref: <https://prometheus.io/docs/practices/naming/#metric-names>

Reply:
Hey @hhromic, yeah it should. 👍 The PR is not ready for a review anyway, I'm testing so far. I'll include the name in the PR. 😉

Reply:
Oh, apologies. Didn't mean to be nosy, my comment was just a small nitpick just in case.

Reply:
Hey @hhromic, no worries, it's all good! 😃👍